packages/module/src/EmptyDetailsView/EmptyDetailsView.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateActions,
+  EmptyStateFooter,
+  EmptyStateProps
+} from '@patternfly/react-core';
+
+export interface EmptyDetailsViewProps extends Omit<EmptyStateProps, 'children' | 'titleText'> {
+  /** Empty state title text. */
+  title?: string;
+  /** Empty state description - will be rendered inside EmptyStateBody. */
+  description?: React.ReactNode;
+  /** Source path to an icon image. */
+  iconImage?: string;
+  /** Alternative text for icon image if image can't load. */
+  imageAlt?: string;
```

Comment:
Do we need alt text for this image? This image is purely decorative and doesn't provide any additional meaning to the user that isn't already provided with the contents on the page.